cnxeasybake/scripts/main.py

```diff
@@ -22,6 +23,12 @@ def easybake(css_in, html_in=sys.stdin, html_out=sys.stdout, last_step=None):
     # serialize out HTML
     print (etree.tostring(html_doc, method="html"), file=html_out)
 
+    # generate CSS coverage_file file
+    if coverage_file:
+        print('SF:{}'.format(css_in.name), file=coverage_file)
```

Comment:
why? (the abspath)
Re: lines, the source_line_end isn't enough, since you can do something like:
```
<ul>, <ol>,
<div data-class="listitem"> { …
```